### ISSUE TITLE
cannot select the found files, and then move/delete them

### ISSUE THREAD

**Issue by CapSel** [nextcloud/server#28483]
It seems that this functionality was lost somewhere/sometime ago.

I'm using newest stable nextcloud.

Pressing ctrl+f allows me to enter file names in "popup dialog" but there is no way from there to select found files to do anything with them. pressing enter takes me to directory of (first) found file and allows me to select this one file... List of files is not filtered. "popup" dialog does not offer tick-boxes.

My use case is: select "copies" of "few" images made in the process of uploading or converting and move them somewhere else. I've got 1000 files in folder and about 300 needs to be moved.

Making tags or flows seems like overkill.

So to sum up:
How do I search whole nextcloud subdirectory for files matching regex (or glob) and move (delete) those files to another directory using only web browser?

**Comment by szaimen** [comment]
Hi, please update to 24.0.9 or better 25.0.3 and report back if it fixes the issue. Thank you!

My goal is to add a label like e.g. 25-feedback to this ticket of an up-to-date major Nextcloud version where the bug could be reproduced. However this is not going to work without your help. So thanks for all your effort!

If you don't manage to reproduce the issue in time and the issue gets closed but you can reproduce the issue afterwards, feel free to create a new bug report with up-to-date information by following this link: https://github.com/nextcloud/server/issues/new?assignees=&labels=bug%2C0.+Needs+triage&template=BUG_REPORT.yml&title=%5BBug%5D%3A+
